src/views/billing/createOrg.vue

```diff
@@ -145,6 +145,127 @@ const stripeGroupsErrors = {
     'try_again_later',
     'transaction_not_allowed',
     'stop_payment_order',
+    'account_country_invalid_address',
```

Comment:
I think we can put it in a static archive

Reply:
yes! we can do it.